doc/src/pair_sw.rst

```diff
@@ -44,6 +48,10 @@ Examples
    pair_style sw/mod maxdelcs 0.25 0.35
    pair_coeff * * tmd.sw.mod Mo S S
 
+   pair_style hybrid sw sw threebody off
+   pair_coeff * * mW_xL.sw mW NULL
+   pair_coeff * 2 mW_xL.sw mW xL
+
```

Comment:
This example is using incorrect syntax. Please see my post at issue #3297 instead.

Reply:
Resolved by e8c2dcc